pkg/systemd/quadlet/quadlet.go

```diff
@@ -876,6 +876,21 @@ func ConvertContainer(container *parser.UnitFile, isUser bool, unitsInfoMap map[
 	return service, nil
 }
 
+func defaultServiceGroup(service *parser.UnitFile, remainAfterExit bool) {
```

Comment:
Sorry for the nit, but please rename this function to something like `defaultOneShotServiceGroup` to differentiate it from the use cases that don't call it